presto-main/src/main/java/com/facebook/presto/operator/ByteArrayUtils.java

```diff
@@ -13,21 +13,18 @@
  */
 package com.facebook.presto.operator;
 
-import io.airlift.slice.Slice;
 import io.airlift.slice.SliceOutput;
 import sun.misc.Unsafe;
 
 import java.lang.reflect.Field;
 
-import static com.google.common.base.Preconditions.checkArgument;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static sun.misc.Unsafe.ARRAY_BYTE_BASE_OFFSET;
-import static sun.misc.Unsafe.ARRAY_INT_INDEX_SCALE;
 import static sun.misc.Unsafe.ARRAY_LONG_INDEX_SCALE;
 
-public class ByteArrayUtils
+public final class ByteArrayUtils
```

Comment:
Is this a duplicate of `io.airlift.slice.ByteArrays` with `checkIndexLength` calls removed? If so, would it make sense to rename it to `UnsafeByteArrays` and add a comment that references `io.airlift.slice.ByteArrays` and explains that `checkIndexLength` calls were removed to improve performance? Then, I'd keep the method signatures the same as in `io.airlift.slice.ByteArrays` or add `Unsafe` suffix:

```
public static void setLong(byte[] bytes, int index, long value)
```

or 

```
public static void setLongUnsafe(byte[] bytes, int index, long value)
```